### PR TITLE
Refactor bridge setup

### PR DESCRIPTION
**Pull request by Luap99**
We should setup the bridge and firewall rules in the same loop. We also
have to check the given network driver. For now we only support bridge
but we can easily add more in the future.

### CODE REVIEW

**Comment by Luap99** [comment]
@baude @flouthoc @mheon PTAL this is a just small cleanup

File: src/commands/setup.rs [hunk 1]
```diff
@@ -46,18 +48,43 @@ impl Setup {
...
         for (net_name, network) in network_options.network_info {
-            debug!("Setting up network {} firewall rules", net_name);
+            debug!(
+                "Setting up network {} with driver {}",
+                net_name, network.driver
+            );
 
-            // Setup basic firewall rules for each network.
-            firewall_driver.setup_network(network)?;
+            match network.driver.as_str() {
```
Comment (mheon) [inline src/commands/setup.rs:60]: I'd love to get this behind an interface eventually similar to what I'm doing with the firewall driver

**Comment by mheon** [comment]
LGTM. I have ideas for future cleanups, but this looks like a good improvement

**Comment by flouthoc** [comment]
LGTM and I'd agree we can move all the logic from `src/command/setup.rs` into a new interface maybe `src/podman/setup.rs`.

**Pull request merged**